go/src/koding/klientctl/config.go

```diff
@@ -54,3 +58,82 @@ func createFolderAtHome(cf ...string) (string, error) {
 
 	return folderName, nil
 }
+
+var b = &object.Builder{
+	Tag:       "json",
+	Sep:       ".",
+	Recursive: true,
+}
+
+func ConfigShow(c *cli.Context, log logging.Logger, _ string) (int, error) {
+	cfg := config.Konfig
+
+	if !c.Bool("defaults") {
+		cfg = &konfig.Konfig{}
+
+		db := konfig.NewCache(konfig.KonfigCache)
+		defer db.Close()
+
+		if err := db.GetValue("konfig", cfg); err != nil && err != storage.ErrKeyNotFound {
+			return 1, err
+		}
+	}
+
+	enc := json.NewEncoder(os.Stdout)
+	enc.SetIndent("", "\t")
+
+	if err := enc.Encode(cfg); err != nil {
+		return 1, err
+	}
+
+	return 0, nil
+}
+
+func ConfigSet(c *cli.Context, log logging.Logger, _ string) (int, error) {
+	if len(c.Args()) != 2 {
+		cli.ShowCommandHelp(c, "set")
+		return 1, nil
+	}
+
+	if err := updateKonfigCache(c.Args().Get(0), c.Args().Get(1)); err != nil {
+		fmt.Println(err)
```

Comment:
Oh I see, so maybe is it better to log to stderr like you do [here](https://github.com/koding/koding/pull/9182/files#diff-47e4ab0f8167fbde5d6b9169cfec58a8R35)? I was misled by `log logging.Logger` in function arguments since is is not underscored.